### PR TITLE
 modify the code with the oldest supported version && modify build_chain with the latest version

### PR DESCRIPTION
**Pull request by cyjseagull**

### CODE REVIEW

File: libconfig/GlobalConfigure.h [hunk 1]
```diff
@@ -50,10 +50,15 @@ class GlobalConfigure
         {
             m_version = RC2_VERSION;
         }
-        else
+        else if (dev::stringCmpIgnoreCase(versionStr, "latest") == 0)
         {
             m_version = LATEST_VERSION;
         }
+        /// default is RC1 version
```
Comment (bxq2011hust) [inline libconfig/GlobalConfigure.h]: `// default is RC1`

File: tools/build_chain.sh [hunk 2]
```diff
@@ -897,7 +898,7 @@ if [ -z ${docker_mode} ];then
         echo "Binary check passed."
     fi
 fi
-
+latest_version=$(${bin_path} --version | grep "FISCO-BCOS Version" | cut -d':' -f2 | sed s/[[:space:]]//g)
```
Comment (bxq2011hust) [inline tools/build_chain.sh]: make use of `version=$(curl -s https://raw.githubusercontent.com/FISCO-BCOS/FISCO-BCOS/master/release_note.txt | sed "s/^[vV]//")`.
change to `latest_version =$(curl -s https://raw.githubusercontent.com/FISCO-BCOS/FISCO-BCOS/master/release_note.txt | sed "s/^[vV]//")`
make `latest_version` global

File: libconfig/GlobalConfigure.h [hunk 3]
```diff
@@ -50,10 +50,15 @@ class GlobalConfigure
         {
             m_version = RC2_VERSION;
         }
-        else
+        else if (dev::stringCmpIgnoreCase(versionStr, "latest") == 0)
```
Comment (bxq2011hust) [inline libconfig/GlobalConfigure.h]: what's the `latest` mean?

Comment (cyjseagull) [reply libconfig/GlobalConfigure.h]: `latest` has been removed

File: tools/build_chain.sh (hunk 2 above)
Comment (cyjseagull) [reply tools/build_chain.sh]: `latest_version` is global. Pls modify this shell script according to your opinion. And modify the wiki.

**Pull request merged**